providers/linux/host_linux.go

```diff
@@ -138,7 +141,7 @@ func (h *host) CPUTime() (types.CPUTimes, error) {
 }
 
 func newHost(fs procFS) (*host, error) {
-	stat, err := fs.NewStat()
+	stat, err := fs.Stat()
```

Comment:
I assume you changed this since `NewStat()` is marked as deprecated---in which case, there's a few more instances if that you might want to change